web/src/lib/components/album-page/album-card.svelte

```diff
@@ -61,35 +63,39 @@
 </script>
 
 <div
-	class="hover:cursor-pointer mt-4 relative"
+	class="hover:cursor-pointer mt-4 border-2 border-transparent dark:hover:border-gray-300 hover:border-black rounded-xl p-5"
 	on:click={() => dispatchClick('click', album)}
 	on:keydown={() => dispatchClick('click', album)}
 	data-testid="album-card"
+	on:mouseenter={() => (imageOverlayElement.style.background = 'rgba(66,80,175,0.25)')}
+	on:mouseleave={() => (imageOverlayElement.style.background = 'transparent')}
 >
 	<!-- svelte-ignore a11y-click-events-have-key-events -->
 	<div
 		id={`icon-${album.id}`}
-		class="absolute top-2 right-2"
+		class="absolute top-6 right-6"
 		on:click|stopPropagation|preventDefault={showAlbumContextMenu}
 		data-testid="context-button-parent"
 	>
-		<CircleIconButton logo={DotsVertical} size={'20'} hoverColor={'rgba(95,99,104, 0.5)'} />
+		<CircleIconButton logo={DotsVertical} size={'20'} hoverColor={'rgb(95,99,104)'} />
 	</div>
 
-	<div class={`aspect-square z-[-1]`}>
+	<div class={`aspect-square relative`}>
 		<img
 			src={imageData}
 			alt={album.id}
 			class={`object-cover h-full w-full transition-all z-0 rounded-xl duration-300 hover:shadow-lg`}
 			data-testid="album-image"
 			draggable="false"
 		/>
+		<div class="w-full h-full absolute top-0 rounded-xl" bind:this={imageOverlayElement} />
```

Comment:
There's no need to use a component binding. You can use `style:background` or preferably css classes like this:

```html
<div id="album-card" class="group ...">
	<div>
		<img />
		<div class="group-hover:bg-indigo-800/20 ..." />
	</div>
</div>
```

Reply:
My gosh I didn't know we can refer parent element in tailwind like this! Muchhhh appreciate you for poiting out this! You are a wizard!